### ISSUE TITLE
add projectID is a filter for data

### ISSUE THREAD

**Issue by sformel-usgs** [gbif/hosted-portals#282]
_TL;DR request to add projectID as a filter for the data in the hosted portals._

The dashboard feature that was shared at today's hosted portal meeting is very cool!  We discussed how it would be useful for users to create dashboards that summarize custom groups of datasets. For example, a project might include multiple datasets that are published by different publishers, IPTs, etc.  It would be very useful for a naive GBIF user (e.g. upper management in an institution) to be able to quickly search by projectID. Or, for a manager to create a dashboard that automatically pulls in all relevant datasets, without the burden of finding all the dataseyKeys first.

This is an equivalent query: https://www.gbif.org/occurrence/search?advanced=1&project_id=NCRMP
And OBIS has landing pages that are approximately what I'm looking to create: https://obis.org/institute/23174

**Comment by MortenHofft** [comment]
There was more interest in this than I had guessed (I had added the option as an experiment mostly as it seemed useful). I'll make some examples of what can be done. I believe above example is already covered.

**Comment by sformel-usgs** [comment]
Yes, this is one version of what I was looking for. But I am also interested in allowing users to build their own dashboards (like you demonstrated) with projectID as one of the filters. Is that possible?

**Comment by MortenHofft** [comment]
I haven't added projectId as a filter in the UI, but that should be a simple addition

**Comment by sformel-usgs** [comment]
Thanks for getting that done so quickly! Looking forward to using it.

**Comment by MortenHofft** [comment]
Available in your [staging env now](https://hp-bison.gbif-staging.org/data/?layout=W1t7ImlkIjoiMG1iZmsiLCJwIjp7fSwidHJhbnNsYXRpb24iOiJkYXNoYm9hcmQucmljaG5lc3MiLCJ0IjoiZGF0YVF1YWxpdHkifSx7ImlkIjoiODVqcnkiLCJwIjp7fSwidHJhbnNsYXRpb24iOiJkYXNoYm9hcmQuc3RhdGlzdGljcyIsInQiOiJvY2N1cnJlbmNlU3VtbWFyeSJ9XSxbeyJpZCI6InFibXhxIiwicCI6eyJ2aWV3IjoiUElFIn0sInQiOiJtb250aCJ9XSxbeyJpZCI6IjVtd2llIiwicCI6eyJ2aWV3IjoiVElNRSJ9LCJ0IjoieWVhciJ9LHsiaWQiOiJkOHU4dyIsInAiOnsidmlldyI6IlRBQkxFIn0sInQiOiJkYXRhc2V0S2V5In1dXQ%3D%3D&projectId=ncrmp&view=DASHBOARD) . I'm closing this issue, feel free to create a new one if needed
